templates/deploy-openstack.tpl

```diff
@@ -83,16 +89,11 @@ sed -i 's/# kolla_enable_tls_internal: true/kolla_enable_tls_internal: true/g' $
 cat $KAYOBE_CONFIG_PATH/environments/$KAYOBE_ENVIRONMENT/kolla/globals-tls-config.yml >> $KAYOBE_CONFIG_PATH/environments/$KAYOBE_ENVIRONMENT/kolla/globals.yml
 
 # Create vault configuration for barbican
-cat << EOF >> $KAYOBE_CONFIG_PATH/environments/$KAYOBE_ENVIRONMENT/secrets.yml
----
-secrets_barbican_approle_secret_id: $(uuidgen)
-EOF
+sed -i "s/secret_id:.*/secret_id: $(uuidgen)/g" KAYOBE_CONFIG_PATH/environments/$KAYOBE_ENVIRONMENT/secrets.yml
```

Comment:
Isn't the secrets.yml file encrypted at this point?

Reply:
So it is. I thought I had added a decrypt line before it but apparently not. I'll push a fix

Reply:
https://github.com/stackhpc/terraform-kayobe-multinode/pull/30